src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/controllers/GenerateSettlementAgreementCopyTest.java

```diff
@@ -68,6 +76,8 @@ public void shouldReturnServerErrorWhenPdfGenerationFails() throws Exception {
 
     private ResultActions makeRequest(String externalId) throws Exception {
         return webClient
-            .perform(get("/documents/settlementAgreement/" + externalId));
+            .perform(get("/documents/settlementAgreement/" + externalId)
+                .header(HttpHeaders.AUTHORIZATION, AUTHORISATION_TOKEN)
+            );
```

Comment:
The same bit of code was added a number of times in this PR. Perhaps we could do something to avoid duplication? What do you think about extracting some helper or moving this to the base class?

Reply:
will move to base class